master/reference/repo-structure.md

```diff
@@ -1,69 +1,63 @@
+| `calico/test` | Docker Image | Contains useful shared testing framework code and dependencies for running unit and system tests of Calico in Docker. |
+| `calico/build` | Docker Image | Build image which includes libcalico and the necessary python tooling to produce binaries from python source code. |
+
+## [libcalico-go](https://github.com/projectcalico/libcalico-go)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| libcalico-go | golang library | Contains a variety of helper methods and classes for managing Calico's data, performing IP address management, and providing useful namespace utilities to manipulate container interfaces. |
+
+## Orchestrator Plugins
 
 There are several integrations available for Calico in a containerized
 environment.  The repositories below hold the plugin code for these
 integrations.
 
- - [calico-mesos](https://github.com/projectcalico/calico-mesos): Implements
-   the Calico plugin for running Calico with the [mesos](https://github.com/apache/mesos)
-   orchestrator.  This plugin may be installed manually (from a binary attached
-   to a relase), or as an RPM which can be created.
-
- - [calico-cni](https://github.com/projectcalico/calico-cni): Implements the
-   Calico plugin for running Calico with any orchestrator that uses the
-   [Container Network Interface](https://github.com/appc/cni), including [rkt](https://github.com/coreos/rkt)
-   and [Kubernetes](https://github.com/kubernetes/kubernetes)
-
- - [libnetwork-plugin](https://github.com/projectcalico/libnetwork-plugin):
-   Implements Calico plugin support for the (libnetwork based) Docker
-   networking plugin.  It provides both network and IPAM drivers which may
-   be used when creating networks through Docker. When calico/node is started with the
-   `calicoctl node run` command, it will also run the libnetwork plugin inside the same
-   container by default.
+## [calico-cni](https://github.com/projectcalico/calico-cni)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `calico` | binary | Calico networking plugin for any orchestrator that uses the [Container Network Interface](https://github.com/appc/cni), e.g. [rkt](https://github.com/coreos/rkt), [Kubernetes](https://github.com/kubernetes/kubernetes), and [Mesos](https://github.com/apache/mesos). |
+| `calico-ipam` | binary | Calico CNI IP address management plugin. |
+
+## [libnetwork-plugin](https://github.com/projectcalico/libnetwork-plugin)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `libnetwork-plugin` | binary | Implements Calico plugin support for the (libnetwork based) Docker networking plugin.  It provides both network and IPAM drivers which may be used when creating networks through Docker. When calico/node is started with the `calicoctl node run` command, it will also run the libnetwork plugin inside the same container by default. |
```

Comment:
> Implements Calico plugin support for the (libnetwork based) Docker networking plugin.

This is a mouthful! 

What about "Docker networking plugin for use with Docker + Docker Swarm"

> When calico/node is started with the `calicoctl node run` command, it will also run the libnetwork plugin inside the same container by default

Not convinced we care about going into this sort of detail in this doc.  I'd just remove.